lms/js_config_types.py

```diff
@@ -4,7 +4,14 @@
 Making this a top level module to avoid circular dependency problems.
 """
 
-from typing import NotRequired, TypedDict
+from typing import NotRequired, TypeAlias, TypedDict
+
+URLTemplate: TypeAlias = str
```

Comment:
I reckon I saw something like this in the FE.

Reply:
I think I ended up removing it 😅